tests/app/shared/issue/duplicatedIssues/duplicated-issues.component.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { DuplicatedIssuesComponent } from '../../../../../src/app/shared/issue/duplicatedIssues/duplicated-issues.component';
+import { DebugElement } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { IssueService } from '../../../../../src/app/core/services/issue.service';
+import { ErrorHandlingService } from '../../../../../src/app/core/services/error-handling.service';
+import { PhaseService } from '../../../../../src/app/core/services/phase.service';
+import { PermissionService } from '../../../../../src/app/core/services/permission.service';
+import { Observable, of } from 'rxjs';
+import { Issue } from '../../../../../src/app/core/models/issue.model';
+import { TEAM_4 } from '../../../../constants/data.constants';
+import {
+    ISSUE_PENDING_MODERATION,
+    ISSUE_PENDING_MODERATION_HIGH_SEVERITY_FEATURE_FLAW,
+    ISSUE_PENDING_MODERATION_LOW_SEVERITY_DOCUMENTATION_BUG
+    } from '../../../../constants/githubissue.constants';
+import { By, HAMMER_LOADER } from '@angular/platform-browser';
+import { MaterialModule } from '../../../../../src/app/shared/material.module';
+import { RouterTestingModule } from '@angular/router/testing';
+import { ApolloTestingModule } from 'apollo-angular/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
+import { Phase } from '../../../../../src/app/core/models/phase.model';
+
+describe('DuplicatedIssuesComponent', () => {
+    let component: DuplicatedIssuesComponent;
+    let debugElement: DebugElement;
+    let nativeElement: HTMLElement;
+    let fixture: ComponentFixture<DuplicatedIssuesComponent>;
+    let dummyDuplicatedIssues: Observable<Issue[]>;
+    let thisIssue: Issue;
+
+    const dummyTeam = TEAM_4;
+    const firstDummyIssue =  Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION_HIGH_SEVERITY_FEATURE_FLAW, dummyTeam);
+    const secondDummyIssue = Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION_LOW_SEVERITY_DOCUMENTATION_BUG, dummyTeam);
+
+    const permissionService: any = jasmine.createSpyObj('PermissionService', ['isTeamResponseEditable', 'isTutorResponseEditable']);
+    const issueService: any = jasmine.createSpyObj('IssueService', ['getDuplicateIssuesFor', 'updateIssueWithComment', 'updateIssue', 'updateLocalStore']);
+    const phaseService: any = jasmine.createSpyObj('PhaseService', [], { currentPhase: Phase.phaseModeration });
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                DuplicatedIssuesComponent
+            ],
+            providers: [
+                IssueService, ErrorHandlingService, PhaseService, PermissionService, {
+                    provide: HAMMER_LOADER,
```

Comment:
Not very sure that this provider is needed, can you clarify whether it's required?

Reply:
I think it was because of this error that was thrown previously: https://github.com/angular/components/issues/14668
I tried removing the import and the tests still run, so I think the error has been resolved by other imports. I have removed this import! Thanks for pointing it out.

Reply:
Actually, I realised that the warning is still shown after the tests finish running. 
![image](https://user-images.githubusercontent.com/66008784/118518338-98a07100-b76a-11eb-9582-7fb4a7d7746a.png)
Should I keep the import or do I still remove it?

Reply:
I see, thanks for the clarification. We'll need to keep it, then.